db_patches/0000_Init.sql

```diff
@@ -17,7 +17,7 @@ BEGIN
       , created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
       , updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
       , booking_type varchar(30) NOT NULL
-      , scheduled_from TIMESTAMP NOT NULL
+      , starts_at TIMESTAMP NOT NULL
```

Comment:
I know I should use db patch for renaming fields, but because these are basically the first steps I think it makes sense to just change it here